api/src/pdc/providers/superstruct/shared/index.ts

```
@@ -1,5 +1,13 @@
-import { coerce, date, enums, integer, pattern, size, string } from "@/lib/superstruct/index.ts";
-
+import { coerce, date, enums, integer, pattern, size, string, Struct, union } from "@/lib/superstruct/index.ts";
+export const CoerceNumberMinMax = (type: Struct<number, null>, min: number, max: number) => {
+  return coerce(size(type, min, max), string(), (v) => {
+    const parsed = parseInt(v, 10);
+    if (isNaN(parsed)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)